packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -67,7 +75,7 @@ export type CallingHandlers = {
     userId: string,
     options?: VideoStreamOptions
   ) => Promise<void | CreateVideoStreamViewResult>;
-  onRemoveParticipant: (userId: string) => Promise<void>;
+  onRemoveParticipant: RemoveParticipantHandler;
```

Comment:
Wondering if there is concerns of using a 2 lines overload declaration like:
onRemoveParticipant: (userId: string) => Promise<void>;
onRemoveParticipant: (userId: CommunicationIdentifier) => Promise<void>;

This looks more understandable for me when I am looking at api file

Reply:
Unfortunately, this syntax doesn't work unless you are implementing the functions in a class :/

Reply:
Good point.
Had to change CallingHandlers type of Interface to make this work.
LMKWYT